src/org/zaproxy/zap/extension/websocket/ZapAddOn.xml

```diff
@@ -7,7 +7,7 @@
 	<url></url>
 	<changes>
 	<![CDATA[
-	Updated for 2.7.0.<br>
+	Do not set messages when switching views if text view shows an error message (Issue 4108)<br>
```

Comment:
It looks like v14 was released, so you should increment the version as well.

https://github.com/zaproxy/zap-extensions/releases

Reply:
Updated now